src/oid.rs

```diff
@@ -16,32 +16,24 @@ impl Oid {
         Oid(s.to_owned())
     }
 
-    /// Convert the OID to a string representation.
-    /// The string contains the IDs separated by dots, for ex: "1.2.840.113549.1.1.5"
-    pub fn to_string(&self) -> String {
```

Comment:
Why was this function removed? Is it related to clippy?

Reply:
Yes. `to_string` is a provided method of `Display`, so it hasn’t been removed—just moved.

Reply:
Right, thanks for the reply